e2e-tests/trailing-slash/cypress/integration/always.js

```diff
@@ -62,26 +72,16 @@ describe(`always`, () => {
     cy.getTestElement(`query-param-hash`).click()
     cy.waitForRouteChange().assertRoute(`/page-2/?query_param=hello#anchor`)
   })
-  it(`client-only-simple without slash`, () => {
+  it(`client-only without slash`, () => {
     cy.getTestElement(`client-only-simple-without`).click()
     cy.waitForRouteChange().assertRoute(`/client-only/without/`)
     cy.getTestElement(`title`).should(`have.text`, `without`)
   })
-  it(`client-only-simple with slash`, () => {
+  it(`client-only with slash`, () => {
```

Comment:
We'll need tests for link visits, too. So you'll want to add links to the index page, copy a test like this and then assert that `title` test element has e.g. `with/with`